src/Flash/Flash.tsx

```diff
@@ -71,8 +73,26 @@ const StyledFlash = styled.div<StyledFlashProps>`
 
 export type FlashProps = ComponentProps<typeof StyledFlash>
 
-const Flash = React.forwardRef(function Flash({as, variant = 'default', ...rest}, ref) {
-  return <StyledFlash ref={ref} as={as} variant={variant} {...rest} />
+const Flash = React.forwardRef(function Flash({as, children, variant = 'default', renderDefaultVisual, ...rest}, ref) {
+  const defaultLeadingVisuals = {
```

Comment:
Nit: should we move this const object out to the module level?

Reply:
Yea, I like that